Stellarium 1.0 fails to build with openGL ES and Qt6

### Expected Behaviour
Stellarium 1.0 should build with openGL ES and Qt6.


### Actual Behaviour
Build fails on openSUSE Tubmleweed aarch64 with openGL ES and Qt6.
```
[  585s] In file included from /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:22:
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/GLFuncs.hpp:32:1: error: expected class-name before '{' token
[  585s]    32 | {
[  585s]       | ^
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/GLFuncs.hpp: In member function 'void GLExtFuncs::init(QOpenGLContext*)':
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/GLFuncs.hpp:42:25: error: 'initializeOpenGLFunctions' was not declared in this scope
[  585s]    42 |                         initializeOpenGLFunctions();
[  585s]       |                         ^~~~~~~~~~~~~~~~~~~~~~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp: In member function 'void Frustum::drawFrustum() const':
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:184:21: error: 'class GLExtFuncs' has no member named 'glColor3f'
[  585s]   184 |         glExtFuncs->glColor3f(0.0f, 0.0f, 1.0f);
[  585s]       |                     ^~~~~~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:186:21: error: 'class GLExtFuncs' has no member named 'glBegin'
[  585s]   186 |         glExtFuncs->glBegin(GL_LINE_LOOP);
[  585s]       |                     ^~~~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:188:29: error: 'class GLExtFuncs' has no member named 'glVertex3f'
[  585s]   188 |                 glExtFuncs->glVertex3f(ntl.v[0],ntl.v[1],ntl.v[2]);
[  585s]       |                             ^~~~~~~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:189:29: error: 'class GLExtFuncs' has no member named 'glVertex3f'
[  585s]   189 |                 glExtFuncs->glVertex3f(ntr.v[0],ntr.v[1],ntr.v[2]);
[  585s]       |                             ^~~~~~~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:190:29: error: 'class GLExtFuncs' has no member named 'glVertex3f'
[  585s]   190 |                 glExtFuncs->glVertex3f(nbr.v[0],nbr.v[1],nbr.v[2]);
[  585s]       |                             ^~~~~~~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:191:29: error: 'class GLExtFuncs' has no member named 'glVertex3f'
[  585s]   191 |                 glExtFuncs->glVertex3f(nbl.v[0],nbl.v[1],nbl.v[2]);
[  585s]       |                             ^~~~~~~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:192:21: error: 'class GLExtFuncs' has no member named 'glEnd'
[  585s]   192 |         glExtFuncs->glEnd();
[  585s]       |                     ^~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:194:21: error: 'class GLExtFuncs' has no member named 'glBegin'
[  585s]   194 |         glExtFuncs->glBegin(GL_LINE_LOOP);
[  585s]       |                     ^~~~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:196:29: error: 'class GLExtFuncs' has no member named 'glVertex3f'
[  585s]   196 |                 glExtFuncs->glVertex3f(ftr.v[0],ftr.v[1],ftr.v[2]);
[  585s]       |                             ^~~~~~~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:197:29: error: 'class GLExtFuncs' has no member named 'glVertex3f'
[  585s]   197 |                 glExtFuncs->glVertex3f(ftl.v[0],ftl.v[1],ftl.v[2]);
[  585s]       |                             ^~~~~~~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:198:29: error: 'class GLExtFuncs' has no member named 'glVertex3f'
[  585s]   198 |                 glExtFuncs->glVertex3f(fbl.v[0],fbl.v[1],fbl.v[2]);
[  585s]       |                             ^~~~~~~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:199:29: error: 'class GLExtFuncs' has no member named 'glVertex3f'
[  585s]   199 |                 glExtFuncs->glVertex3f(fbr.v[0],fbr.v[1],fbr.v[2]);
[  585s]       |                             ^~~~~~~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:200:21: error: 'class GLExtFuncs' has no member named 'glEnd'
[  585s]   200 |         glExtFuncs->glEnd();
[  585s]       |                     ^~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:202:21: error: 'class GLExtFuncs' has no member named 'glBegin'
[  585s]   202 |         glExtFuncs->glBegin(GL_LINE_LOOP);
[  585s]       |                     ^~~~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:204:29: error: 'class GLExtFuncs' has no member named 'glVertex3f'
[  585s]   204 |                 glExtFuncs->glVertex3f(nbl.v[0],nbl.v[1],nbl.v[2]);
[  585s]       |                             ^~~~~~~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:205:29: error: 'class GLExtFuncs' has no member named 'glVertex3f'
[  585s]   205 |                 glExtFuncs->glVertex3f(nbr.v[0],nbr.v[1],nbr.v[2]);
[  585s]       |                             ^~~~~~~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:206:29: error: 'class GLExtFuncs' has no member named 'glVertex3f'
[  585s]   206 |                 glExtFuncs->glVertex3f(fbr.v[0],fbr.v[1],fbr.v[2]);
[  585s]       |                             ^~~~~~~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:207:29: error: 'class GLExtFuncs' has no member named 'glVertex3f'
[  585s]   207 |                 glExtFuncs->glVertex3f(fbl.v[0],fbl.v[1],fbl.v[2]);
[  585s]       |                             ^~~~~~~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:208:21: error: 'class GLExtFuncs' has no member named 'glEnd'
[  585s]   208 |         glExtFuncs->glEnd();
[  585s]       |                     ^~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:210:21: error: 'class GLExtFuncs' has no member named 'glBegin'
[  585s]   210 |         glExtFuncs->glBegin(GL_LINE_LOOP);
[  585s]       |                     ^~~~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:212:29: error: 'class GLExtFuncs' has no member named 'glVertex3f'
[  585s]   212 |                 glExtFuncs->glVertex3f(ntr.v[0],ntr.v[1],ntr.v[2]);
[  585s]       |                             ^~~~~~~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:213:29: error: 'class GLExtFuncs' has no member named 'glVertex3f'
[  585s]   213 |                 glExtFuncs->glVertex3f(ntl.v[0],ntl.v[1],ntl.v[2]);
[  585s]       |                             ^~~~~~~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:214:29: error: 'class GLExtFuncs' has no member named 'glVertex3f'
[  585s]   214 |                 glExtFuncs->glVertex3f(ftl.v[0],ftl.v[1],ftl.v[2]);
[  585s]       |                             ^~~~~~~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:215:29: error: 'class GLExtFuncs' has no member named 'glVertex3f'
[  585s]   215 |                 glExtFuncs->glVertex3f(ftr.v[0],ftr.v[1],ftr.v[2]);
[  585s]       |                             ^~~~~~~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:216:21: error: 'class GLExtFuncs' has no member named 'glEnd'
[  585s]   216 |         glExtFuncs->glEnd();
[  585s]       |                     ^~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:218:21: error: 'class GLExtFuncs' has no member named 'glBegin'
[  585s]   218 |         glExtFuncs->glBegin(GL_LINE_LOOP);
[  585s]       |                     ^~~~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:220:29: error: 'class GLExtFuncs' has no member named 'glVertex3f'
[  585s]   220 |                 glExtFuncs->glVertex3f(ntl.v[0],ntl.v[1],ntl.v[2]);
[  585s]       |                             ^~~~~~~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:221:29: error: 'class GLExtFuncs' has no member named 'glVertex3f'
[  585s]   221 |                 glExtFuncs->glVertex3f(nbl.v[0],nbl.v[1],nbl.v[2]);
[  585s]       |                             ^~~~~~~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:222:29: error: 'class GLExtFuncs' has no member named 'glVertex3f'
[  585s]   222 |                 glExtFuncs->glVertex3f(fbl.v[0],fbl.v[1],fbl.v[2]);
[  585s]       |                             ^~~~~~~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:223:29: error: 'class GLExtFuncs' has no member named 'glVertex3f'
[  585s]   223 |                 glExtFuncs->glVertex3f(ftl.v[0],ftl.v[1],ftl.v[2]);
[  585s]       |                             ^~~~~~~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:224:21: error: 'class GLExtFuncs' has no member named 'glEnd'
[  585s]   224 |         glExtFuncs->glEnd();
[  585s]       |                     ^~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:226:21: error: 'class GLExtFuncs' has no member named 'glBegin'
[  585s]   226 |         glExtFuncs->glBegin(GL_LINE_LOOP);
[  585s]       |                     ^~~~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:228:29: error: 'class GLExtFuncs' has no member named 'glVertex3f'
[  585s]   228 |                 glExtFuncs->glVertex3f(nbr.v[0],nbr.v[1],nbr.v[2]);
[  585s]       |                             ^~~~~~~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:229:29: error: 'class GLExtFuncs' has no member named 'glVertex3f'
[  585s]   229 |                 glExtFuncs->glVertex3f(ntr.v[0],ntr.v[1],ntr.v[2]);
[  585s]       |                             ^~~~~~~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:230:29: error: 'class GLExtFuncs' has no member named 'glVertex3f'
[  585s]   230 |                 glExtFuncs->glVertex3f(ftr.v[0],ftr.v[1],ftr.v[2]);
[  585s]       |                             ^~~~~~~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:231:29: error: 'class GLExtFuncs' has no member named 'glVertex3f'
[  585s]   231 |                 glExtFuncs->glVertex3f(fbr.v[0],fbr.v[1],fbr.v[2]);
[  585s]       |                             ^~~~~~~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:232:21: error: 'class GLExtFuncs' has no member named 'glEnd'
[  585s]   232 |         glExtFuncs->glEnd();
[  585s]       |                     ^~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:235:21: error: 'class GLExtFuncs' has no member named 'glBegin'
[  585s]   235 |         glExtFuncs->glBegin(GL_LINES);
[  585s]       |                     ^~~~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:239:29: error: 'class GLExtFuncs' has no member named 'glVertex3f'
[  585s]   239 |                 glExtFuncs->glVertex3f(a.v[0],a.v[1],a.v[2]);
[  585s]       |                             ^~~~~~~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:240:29: error: 'class GLExtFuncs' has no member named 'glVertex3f'
[  585s]   240 |                 glExtFuncs->glVertex3f(b.v[0],b.v[1],b.v[2]);
[  585s]       |                             ^~~~~~~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:245:29: error: 'class GLExtFuncs' has no member named 'glVertex3f'
[  585s]   245 |                 glExtFuncs->glVertex3f(a.v[0],a.v[1],a.v[2]);
[  585s]       |                             ^~~~~~~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:246:29: error: 'class GLExtFuncs' has no member named 'glVertex3f'
[  585s]   246 |                 glExtFuncs->glVertex3f(b.v[0],b.v[1],b.v[2]);
[  585s]       |                             ^~~~~~~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:251:29: error: 'class GLExtFuncs' has no member named 'glVertex3f'
[  585s]   251 |                 glExtFuncs->glVertex3f(a.v[0],a.v[1],a.v[2]);
[  585s]       |                             ^~~~~~~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:252:29: error: 'class GLExtFuncs' has no member named 'glVertex3f'
[  585s]   252 |                 glExtFuncs->glVertex3f(b.v[0],b.v[1],b.v[2]);
[  585s]       |                             ^~~~~~~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:257:29: error: 'class GLExtFuncs' has no member named 'glVertex3f'
[  585s]   257 |                 glExtFuncs->glVertex3f(a.v[0],a.v[1],a.v[2]);
[  585s]       |                             ^~~~~~~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:258:29: error: 'class GLExtFuncs' has no member named 'glVertex3f'
[  585s]   258 |                 glExtFuncs->glVertex3f(b.v[0],b.v[1],b.v[2]);
[  585s]       |                             ^~~~~~~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:263:29: error: 'class GLExtFuncs' has no member named 'glVertex3f'
[  585s]   263 |                 glExtFuncs->glVertex3f(a.v[0],a.v[1],a.v[2]);
[  585s]       |                             ^~~~~~~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:264:29: error: 'class GLExtFuncs' has no member named 'glVertex3f'
[  585s]   264 |                 glExtFuncs->glVertex3f(b.v[0],b.v[1],b.v[2]);
[  585s]       |                             ^~~~~~~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:269:29: error: 'class GLExtFuncs' has no member named 'glVertex3f'
[  585s]   269 |                 glExtFuncs->glVertex3f(a.v[0],a.v[1],a.v[2]);
[  585s]       |                             ^~~~~~~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:270:29: error: 'class GLExtFuncs' has no member named 'glVertex3f'
[  585s]   270 |                 glExtFuncs->glVertex3f(b.v[0],b.v[1],b.v[2]);
[  585s]       |                             ^~~~~~~~~~
[  585s] /home/abuild/rpmbuild/BUILD/stellarium-1.0/plugins/Scenery3d/src/Frustum.cpp:271:21: error: 'class GLExtFuncs' has no member named 'glEnd'
[  585s]   271 |         glExtFuncs->glEnd();
[  585s]       |                     ^~~~~
[  585s] make[2]: *** [plugins/Scenery3d/src/CMakeFiles/Scenery3d-static.dir/build.make:115: plugins/Scenery3d/src/CMakeFiles/Scenery3d-static.dir/Frustum.cpp.o] Error 1
```

### Steps to reproduce
Try to build with openGL ES and Qt6.


### System
* Stellarium version: 1.0
* Operating system:  openSUSE Tumbleweed aarch64
* Graphics Card: N/A
* Screen type (if applicable): N/A

### Logfile 
Log provided above.

--- a/plugins/Scenery3d/src/Frustum.cpp
+++ b/plugins/Scenery3d/src/Frustum.cpp
@@ -170,7 +170,7 @@ void Frustum::drawFrustum() const
 {
 // Minimum to avoid trouble when building on pure OpenGL ES systems
 // Not sure about ANGLE!
-#if !defined(QT_OPENGL_ES_2)
+#if !QT_CONFIG(opengles2)
 
 	Vec3f ntl = drawCorners[NTL];
 	Vec3f ntr = drawCorners[NTR];

--- a/plugins/Scenery3d/src/GLFuncs.hpp
+++ b/plugins/Scenery3d/src/GLFuncs.hpp
@@ -24,7 +24,7 @@
 #include <QOpenGLContext>
 #include <QOpenGLFunctions_1_0>
 
-#ifndef QT_OPENGL_ES_2
+#if !QT_CONFIG(opengles2)
 //! Defines some OpenGL functions not resolved through StelOpenGL (which only contains base OpenGL ES2 functions)
 //! Using the QOpenGLFunctions_*_* directly would solve this better, but it conflicts with the
 //! current StelOpenGL header dramatically.

--- a/plugins/Scenery3d/src/Polyhedron.cpp
+++ b/plugins/Scenery3d/src/Polyhedron.cpp
@@ -254,7 +254,7 @@ const QVector<Vec3f> &Polyhedron::getVerts() const
 
 void Polyhedron::render() const
 {
-#if !defined(QT_OPENGL_ES_2)
+#if !QT_CONFIG(opengles2)
 
 	//render each polygon
 	glExtFuncs->glColor3f(0.4f,0.4f,0.4f);

--- a/plugins/Scenery3d/src/S3DRenderer.cpp
+++ b/plugins/Scenery3d/src/S3DRenderer.cpp
@@ -52,7 +52,7 @@ Q_LOGGING_CATEGORY(s3drenderer, "stel.plugin.scenery3d.renderer")
 static const float LUNAR_BRIGHTNESS_FACTOR=0.2f;
 static const float VENUS_BRIGHTNESS_FACTOR=0.005f;
 
-#ifndef QT_OPENGL_ES_2
+#if !QT_CONFIG(opengles2)
 //this is the place where this is initialized
 GLExtFuncs* glExtFuncs;
 #endif
@@ -120,7 +120,7 @@ S3DRenderer::~S3DRenderer()
 	deleteShadowmapping();
 	deleteCubemapping();
 
-#ifndef QT_OPENGL_ES_2
+#if !QT_CONFIG(opengles2)
 	//delete extension functions
 	delete glExtFuncs;
 #endif
@@ -1403,7 +1403,7 @@ void S3DRenderer::drawCoordinatesText()
 void S3DRenderer::drawDebug()
 {
 	//frustum/box debug rendering only on desktop GL
-#ifndef QT_OPENGL_ES_2
+#if !QT_CONFIG(opengles2)
 	if(!shaderParameters.openglES)
 	{
 		QOpenGLShaderProgram* debugShader = shaderManager.getDebugShader();
@@ -1654,7 +1654,7 @@ void S3DRenderer::init()
 	QOpenGLContext* ctx = QOpenGLContext::currentContext();
 	Q_ASSERT(ctx);
 
-#ifndef QT_OPENGL_ES_2
+#if !QT_CONFIG(opengles2)
 	//initialize additional functions needed and not provided through StelOpenGL
 	glExtFuncs = new GLExtFuncs();
 	glExtFuncs->init(ctx);
@@ -1787,7 +1787,7 @@ bool S3DRenderer::initCubemapping()
 	}
 
 
-#ifndef QT_OPENGL_ES_2
+#if !QT_CONFIG(opengles2)
 	//if we are on an ES context, it may not be possible to specify texture bitdepth
 	bool isEs = QOpenGLContext::currentContext()->isOpenGLES();
 	GLint colorFormat = isEs ? GL_RGBA : GL_RGBA8;
@@ -1916,7 +1916,7 @@ bool S3DRenderer::initCubemapping()
 
 		GET_GLERROR()
 
-#ifndef QT_OPENGL_ES_2
+#if !QT_CONFIG(opengles2)
 		//attach cube tex + cube depth
 		//note that this function will be a NULL pointer if GS is not supported, so it is important to check support before using
 		glExtFuncs->glFramebufferTexture(GL_FRAMEBUFFER, GL_COLOR_ATTACHMENT0,cubeMapCubeTex,0);
@@ -2219,7 +2219,7 @@ bool S3DRenderer::initShadowmapping()
 			//Bind the depth map and setup parameters
 			glBindTexture(GL_TEXTURE_2D, shadowMapsArray.at(i));
 
-#ifndef QT_OPENGL_ES_2
+#if !QT_CONFIG(opengles2)
 			bool isEs = QOpenGLContext::currentContext()->isOpenGLES();
 			GLenum depthPcss = isEs ? GL_DEPTH_COMPONENT : GL_DEPTH_COMPONENT32F;
 			GLenum depthNormal = isEs ? GL_DEPTH_COMPONENT : GL_DEPTH_COMPONENT16;
@@ -2238,7 +2238,7 @@ bool S3DRenderer::initShadowmapping()
 			//NOTE: can't use depth compare mode on ES2
 			if(!pcssEnabled)
 			{
-#ifndef QT_OPENGL_ES_2
+#if !QT_CONFIG(opengles2)
 				if(!isEs)
 				{
 					glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_COMPARE_MODE, GL_COMPARE_R_TO_TEXTURE);
@@ -2254,7 +2254,7 @@ bool S3DRenderer::initShadowmapping()
 					|| shaderParameters.shadowFilterQuality == S3DEnum::SFQ_HIGH_HARDWARE) ? GL_LINEAR : GL_NEAREST;
 			glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_MIN_FILTER, filter);
 			glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_MAG_FILTER, filter);
-#ifndef QT_OPENGL_ES_2
+#if !QT_CONFIG(opengles2)
 			if(!isEs)
 			{
 				glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_BASE_LEVEL,0);
@@ -2273,7 +2273,7 @@ bool S3DRenderer::initShadowmapping()
 			//but the respective functions are not available on GLES2?
 			//On ANGLE, it seems to work without this settings (framebuffer is complete, etc.)
 			//but I don't know if it will work on other ES platforms?
-#ifndef QT_OPENGL_ES_2
+#if !QT_CONFIG(opengles2)
 			if(!isEs)
 			{
 				glExtFuncs->glDrawBuffer(GL_NONE); // essential for depth-only FBOs!!!

--- a/plugins/Scenery3d/src/SPolygon.cpp
+++ b/plugins/Scenery3d/src/SPolygon.cpp
@@ -118,7 +118,7 @@ void SPolygon::addUniqueVert(const Vec3f &v)
 
 void SPolygon::render()
 {
-#if !defined(QT_OPENGL_ES_2)
+#if !QT_CONFIG(opengles2)
 	//render each polygon
 	glExtFuncs->glColor3f(0.4f,0.4f,0.4f);
 

--- a/src/core/StelOpenGL.hpp
+++ b/src/core/StelOpenGL.hpp
@@ -60,7 +60,7 @@ namespace StelOpenGL
 
 // This is still needed for the ARM platform (armhf)
 
-#if defined(QT_OPENGL_ES_2)
+#if QT_CONFIG(opengles2)
 #ifndef GL_DOUBLE
 #define GL_DOUBLE GL_FLOAT
 #endif

--- a/src/core/modules/Planet.cpp
+++ b/src/core/modules/Planet.cpp
@@ -3345,7 +3345,7 @@ bool Planet::initFBO()
 		GL(gl->glActiveTexture(GL_TEXTURE1));
 		GL(gl->glBindTexture(GL_TEXTURE_2D, shadowTex));
 
-#ifndef QT_OPENGL_ES_2
+#if !QT_CONFIG(opengles2)
 		if(!isGLESv2)
 		{
 			GL(gl->glTexParameteri(GL_TEXTURE_2D, GL_TEXTURE_BASE_LEVEL, 0));
@@ -3376,7 +3376,7 @@ bool Planet::initFBO()
 		//see GL_EXT_framebuffer_object and GL_ARB_framebuffer_object
 		//on ES 2, this seems to be allowed (there are no glDrawBuffers/glReadBuffer functions there), see GLES spec section 4.4.4
 		//probably same on ES 3: though it has glDrawBuffers/glReadBuffer but no mention of it in section 4.4.4 and no FRAMEBUFFER_INCOMPLETE_DRAW_BUFFER is defined
-#ifndef QT_OPENGL_ES_2
+#if !QT_CONFIG(opengles2)
 		if(!ctx->isOpenGLES())
 		{
 #if (QT_VERSION>=QT_VERSION_CHECK(6,0,0))

